pandas/src/joins_func_helper.pxi.in

```diff
@@ -158,3 +163,21 @@ def asof_join_{{on_dtype}}(ndarray[{{on_dtype}}] left_values,
 
 {{endfor}}
 
+
+#----------------------------------------------------------------------
+# stringify
+#----------------------------------------------------------------------
+
+def stringify(ndarray[object, ndim=2] xt):
+    cdef:
+        Py_ssize_t n
+        ndarray[object] result
+
+    n = len(xt)
+    result = np.empty(n, dtype=np.object)
+
+    for i in range(n):
+        result[i] = xt[i].tostring()
```

Comment:
oh if all u need is hashing we just added this: 

https://github.com/pandas-dev/pandas/blob/master/pandas/tools/hashing.py